### ISSUE TITLE
RadiusSum: progress report

### ISSUE THREAD

**Issue by mantid-builder** [mantidproject/mantid#9112]
sans_code
This issue was originally [TRAC 8267](http://trac.mantidproject.org/mantid/ticket/8267)

Original Reporter: Gesner Passos

This ticket is blocked by : 
- [ ] [TRAC8112](http://trac.mantidproject.org/mantid/ticket/8112)

Implement progress report on radiussum algorithm

**Comment by gemmaguest** [comment]
Closing as we have another issue open to investigate this #8376 23976
